File: aemet/SergioDeIscar-aemet-java/SergioDeIscar-aemet-java/src/main/java/org/example/formateadores/Formateador.java
package org.example.formateadores;

import java.time.LocalTime;
import java.time.format.DateTimeFormatter;

public class Formateador {
    public static String toFormatString(LocalTime time){
        return time.format(DateTimeFormatter.ofPattern("HH:mm:ss"));
    }
    public static LocalTime toLocalTimeFormate(String time){
        String[] split = time.split(":");
        return LocalTime.of(Integer.parseInt(split[0]), Integer.parseInt(split[1]), 0);
    }
}

File: aemet/SergioDeIscar-aemet-kotlin/SergioDeIscar-aemet-kotlin/src/main/kotlin/formateadores/Formateador.kt
package formateadores

import java.time.LocalTime
import java.time.format.DateTimeFormatter

fun LocalTime.toFormatString(): String{
    return this.format(DateTimeFormatter.ofPattern("HH:mm:ss"))
}

fun String.toLocalTimeFormate(): LocalTime{
    val split = this.split(":")
    return LocalTime.of(split[0].toInt(), split[1].toInt(), 0)
}